src/storage/src/monitor/monitored_store.rs

```diff
@@ -78,7 +78,7 @@ impl<S> MonitoredStateStore<S> {
             .inspect_err(|e| error!("Failed in iter: {:?}", e))?;
 
         self.storage_metrics
-            .iter_duration
+            .iter_init_duration
```

Comment:
I suppose this new name shows the difference from `iter_scan_duration`?

Reply:
+1

cc. @Li0k